src/gasman.c

```diff
@@ -772,10 +772,8 @@ void InitGlobalBag (
 }
 
 
-static Int IsLessGlobal (
-    const Char *        cookie1,
-    const Char *        cookie2,
-    UInt                byWhat )
+static BOOL
+IsLessGlobal(const Char * cookie1, const Char * cookie2, UInt byWhat)
```

Comment:
A general question here: Above, you changed return values `0` and `1` to `FALSE` and `TRUE`, below in lines 783 and following (I can't mark that line as there is no change in it), you left the return values `0`, `1` and `-1`. Why did you change the return value of the function to BOOL then?

Reply:
This was a mistake, thank you for catching it. I'll restore the return type, and also rename the function (IMHO anything starting with `Is...` like this should return a BOOL)

Reply:
Actually, looking at the code in question closer, I think it simply is buggy (probably due to a past refactoring?) -- indeed, its caller clearly expects it to return a BOOL.

Reply:
I refactored the code in question in PR #3815